src/Components/Modal/Modal.module.css

```diff
@@ -90,18 +91,27 @@
 .modalBody {
   height: 700px;
   background-color: var(--lightgrey);
-  text-align: center;
 }
 
 .createDesignBtn {
   position: absolute;
-  right: 70px;
-  top: 10px;
-  height: 45px;
-  width: 155px;
+  right: 77px;
+  top: 13px;
+  height: 46px;
+  width: 158px;
 }
-/* laptop screen size */
-@media only screen and (max-width: 1024px) {
+
+@media only screen and (max-width: var(--large-screen)) {
```

Comment:
these do work?
again, don't need the `only screen` part. This app won't be printed ;)